src/libcore/cell.rs

```diff
@@ -641,6 +641,8 @@ pub struct UnsafeCell<T: ?Sized> {
     ///
     /// This field should not be accessed directly, it is made public for static
     /// initializers.
+    #[deprecated(since = "1.2.0", reason = "use `get` to access the wrapped value or `new` to
```

Comment:
Also, could you be sure to wrap this line to 80 characters? (the surrounding style)